Gemfile.lock

```diff
@@ -461,6 +476,7 @@ DEPENDENCIES
   aws-sdk-s3
   better_delayed_job_web
   bootsnap (>= 1.1.0)
+  bootstrap4-kaminari-views
```

Comment:
screenshots would have been nice here in the PR for such visual additions.
made one, sooo beautiful:

![Bildschirmfoto 2022-01-12 um 11 40 20](https://user-images.githubusercontent.com/1942953/149125049-8c5259af-54cd-41e9-931e-3a9861173394.png)

Reply:
thx